tuftool/src/source.rs

```diff
@@ -1,176 +1,88 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
+#![allow(clippy::doc_markdown)]
 //! Private keys are generally provided as paths, but may sometimes be provided as a URL. For
-//! example, when one of the Rusoto features is enabled, you can use an aws-ssm:// URL to refer to
-//! a key accessible in SSM.
+//! example, when one of the Rusoto features is enabled, you can use an aws-ssm:// special URL
+//! to refer to a key accessible in SSM. (See below for more format examples)
 //!
 //! This module parses a key source command line parameter as a URL, relative to `file://$PWD`,
 //! then matches the URL scheme against ones we understand.
+//!
+//! Currently supported key sources are local files and AWS SSM.
+//!
+//! Examples of currently supported formats:
+//!
+//! Local files may be specified using a path or "file:///" prefixed path:
+//! "./a/key/file/here"
+//! "file:///./a/key/file/here" (notice the 3 slashes after the colon)
```

Comment:
Is this a quirk of the url library, using three slashes even for a relative path?  Would an absolute path have four?

Reply:
Sort of? Normally an URL would have a host string there, but since there isn't one for files, that portion of the "URL" is blank.